### DISCUSSION TITLE
Scaling parameters from adult to pediatrics

### DISCUSSION THREAD

**Discussion by faisalhhu** [Open-Systems-Pharmacology/Forum#818] [Questions & Problems] [answered]
I have developed an adult PBPK model and scaled it to 6-year-old children. 

I can see the demographics in "characteristics of individual" changed to 6-year-old children. However, when I checked renal and metabolic clearances, it was still using 70kg weight and adult blood flow and liver volumes. It seems that the parameters of the
kidney and liver were not scaled. I have attached the snapshots. 

 
![1 yr old](https://user-images.githubusercontent.com/40292409/95065203-011d6f00-0701-11eb-9bf5-178483e839ff.JPG)

![changes in kidney and liver eliminaion](https://user-images.githubusercontent.com/40292409/95048757-b263db00-06e8-11eb-88e9-b95828760ada.JPG)
Kindly guide if this is normal and ok?

**Comment by msevestre** [comment]
@faisalhhu This is by design. The `calculation parameters` are used to derive the `Specific clearance` value. This value is in theory independent from your individual. Only this parameter will be used in your simulation

**Comment by faisalhhu** [comment]
Thank you for your reply. 
The calculation parameters like volume, blood flow rates of the kidney, etc are the same as adult values. Should not we expect some changes in the calculation parameters in pediatrics e.g. a 1-year-old child compared to adults?

**Comment by faisalhhu** [comment]
In addition, BW is used in the calculation of specific clearance, When we scaled it to a 1-year-old child, the BW of the 1-year-old child should be used in the formula, however, it may be using 73kg in the formula?

**Comment by msevestre** [comment]
@faisalhhu as I said, the only parameter used in the simulation is the CL_spec that is independent of the original individual.
If you look at the formula of your clearance process in your 1year old simulation, you'll see that the value used is 5.32 which was derived from adult parameters.

**Comment by faisalhhu** [comment]
Thank you. Finally, I wanted to understand why it is using adult parameters to calculate specific clearance  CL_spec and not pediatric calculation parameters?

**Comment by faisalhhu** [comment]
if I change BW from 73 (Adult BW) to 10 (1 yr old BW) the specific clearance  "CL_spec" changes. From my point of view, I was expecting to see 10kg as BW when calculating  CL_spec for a 1-year-old child instead of 73kg.

**Comment by msevestre** [comment]
Of course, if you change the BW of adult (the reference individual) then the CL_spec will change. The idea is that the CL_spec is calculated based on the experimental values that you have (For instance Plasma Clearance for a 30 years old adult) and then it can be used in other individuals. If you change the base individual, the CL_spec will change

**Comment by prvmalik** [comment, marked as answer]
@faisalhhu 
The scaling of pediatric clearance in PKSim requires input from the user. It is not automatic.

You are correct - the present parameterization of your model with total hepatic clearance and total renal clearance is only appropriate for adults.  

The following parameters are updated by default for virtual children in PKSim:
- Organ volumes, compositions and blood flows
- Glomerular filtration rate
- Ontogeny factor for binding to albumin or alpha-acid glycoprotein (your selection)

Clearance is scaled with input from the user. In most cases, scaling clearance to children requires knowing which enzyme is responsible for the metabolism (e.g., CYP3A4), and parameterizing your model with these explicit mechanisms. Then, the specific clearance mediated by CYP3A4 can be scaled by the ontogeny factors which are available in PK-Sim (see: "Ontogeny/Variability like" selection when building your individual). 

Scaling renal clearance is usually accomplished by determining the relative contributions between glomerular filtration, tubular secretion and reabsorption (if applicable). The glomerular filtration rate is scaled by default. I would say we do not yet know with certainty how to scale clearance due to tubular secretion, but Kathy Kit-Wun Cheung has some papers on the ontogeny for the relevant transporters, as a start.

I see you are looking for the ontogeny of CES1. I would start with this paper: 10.1124/dmd.116.072652

> Finally, I wanted to understand why it is using adult parameters to calculate specific clearance CL_spec and not pediatric calculation parameters?

Because scaling clearance requires input from the user.

**Reply by huke58** [reply]
@prvmalik  hi, I've seen this discussion recently and just read this article(10.1124/dmd.116.072652). 
I've got some questions.  We can see that the ontogeny equation of CES1 is obtained in this article. I have calculated the F values at different ages using this formula. However, this value does not reach 100% in adulthood (for example, at 30 years old, it is 0.9). How should this be explained?

**Comment by faisalhhu** [comment]
Thank you both for explaining. I shall look more into this. At present, I am modeling the drug and metabolite by using CL info we have in the literature and from the POPPK model we had developed. Also, I was planning to make it a diseased model by changing the blood flow rates, however, I noticed similar flow rates and liver volumes in adults and pediatrics so started to look more at how PKSim is scaling adult values to pediatric values. At present we are assuming the drug and metabolite is eliminated via kidney and there is conversion of the drug to a metabolite (CL of Drug to Met value was added fromPOPPK model) with no biliary CL . I am using renal CL values from POPPK model (Accounted Urine data to calculate renal Cl)  that I think will account for all complex kidney functions.

Thank you all for the explanation.
